server/api/beverages.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const router = express.Router();
+const { Beverage } = require('../db/index');
+
+router.get('/', async (req, res, next) => {
+  try {
+    const beverages = await Beverage.findAll();
+    res.send(beverages);
+  } catch (err) {
+    next(err);
+  }
+});
+
+router.get('/:id', async (req, res, next) => {
+  try {
+    const beverage = await Beverage.findByPK(req.params.id);
+    res.send(beverage);
+  } catch (err) {
+    next(err);
+  }
+});
+
+router.post('/', async (req, res, next) => {
+  //allows only Admin to create new Beverages
+  if (req.isAdmin) {
+    try {
+      const newBeverage = await Beverage.create(req.body);
```

Comment:
instead of req.body, consider sanitizing your data by selecting exactly which fields you want to include - build a new object